### ISSUE TITLE
Inject some Angular in React container

### ISSUE THREAD

**Issue by MichaelRSilva** [dale-waterworth/micro-front-end-container#1]
@dale-waterworth When we try to inject some Angular code into the React container, the error occurs: 

`Expected the result of a dynamic import () call. Instead received: [object Module] '`. 

Were you able to solve this problem?

**Comment by dale-waterworth** [comment]
@MichaelRSilva yeah i got the same when i tried, couldn't figure it out. Seems like it's either one or the other but not react and angular. Please keep me posted if you figure it out :)
